src/SFA.DAS.EAS.Web/Views/EmployerAccountTransactions/TransactionsDownload.cshtml

```diff
@@ -1,29 +1,95 @@
-﻿@using System.Globalization
-@model SFA.DAS.EAS.Web.OrchestratorResponse<SFA.DAS.EAS.Web.ViewModels.TransactionsDownloadResultViewModel>
-@{
-    var culture = new CultureInfo("en-GB");
-}
-@{ViewBag.PageID = "page-transactions"; }
+﻿@using SFA.DAS.EAS.Application.Formatters.TransactionDowloads
+@using SFA.DAS.EAS.Web.Extensions
+@model SFA.DAS.EAS.Web.ViewModels.TransactionDownloadViewModel
+@{ViewBag.PageID = "download-transactions"; }
 @{ViewBag.Section = "finance"; }
 @{ViewBag.Title = "Download transactions";}
-
 @{
     ViewBag.GaData.Vpv = "/accounts/finance/downloadtransactions";
 }
-
+<script type="text/javascript">
+    function clearErrors() {
+        $('.error-summary').hide();
+        $('.field-validation-error').hide();
+        $('#end-date-group').removeClass("error");
+        $('#start-date-group').removeClass("error");
+    }
+</script>
+@Html.Partial("_ValidationSummary", ViewData.ModelState.DistinctByErrorMessage())
 <div class="grid-row">
     <div class="column-full">
         <h1 class="heading-xlarge">
             Download transactions
         </h1>
+
+        <div class="column-one-third">
+            <p>Enter a start and end date to download transactions within a specific date range.</p>
+
+            @using (Html.BeginForm("TransactionDownloadByDate", "EmployerAccountTransactions", FormMethod.Post, new { onsubmit = "return clearErrors();" }))
+    {
+        @Html.AntiForgeryToken()
+        @Html.HiddenFor(m => m.Message.AccountHashedId)
+        <div id="start-date-group" class="form-error-group form-group @(Html.IsValid(m => m.Message.StartDate) ? "" : "error")">
+            <span class="form-label-bold">Start date</span>
+            <span class="form-hint">For example, 5 2017</span>
+            <div id="StartDate" class="form-date">
+                @Html.ValidationMessageFor(m => m.Message.StartDate)
+                <div class="form-group form-group-month">
+                    <label for="StartDate.Month">
+                        Month
+                    </label>
+                    @Html.TextBox("StartDate.Month", Model.StartDate.Month, new { @class = "form-control length-limit", type = "number", maxlength = "2", min = "1", max = "12", aria_labelledby = "StartDate.Month" })
```

Comment:
I recommend using the expression based html helpers to keep the view strongly typed.